### PR TITLE
fixed OurRational sample code

### PR DESCRIPTION
**Pull request by mtomassoli**
Please check my English as I'm not a native speaker.

### CODE REVIEW

File: doc/src/manual/constructors.md [hunk 1]
```diff
@@ -420,7 +420,7 @@ julia> struct OurRational{T<:Integer} <: Real
                if num == 0 && den == 0
                     error("invalid rational: 0//0")
                end
-               g = gcd(den, num)
+               g = sign(den) < 0 ? -gcd(num, den) : gcd(num, den)
```
Comment (StefanKarpinski) [inline doc/src/manual/constructors.md]:
```suggestion
               g = copysign(gcd(num, den), num)
```

Comment (mtomassoli) [reply doc/src/manual/constructors.md]: Do you mean

```suggestion
               g = copysign(gcd(num, den), den)
```
?

Comment (StefanKarpinski) [reply doc/src/manual/constructors.md]: Sure, that makes sense.

**Comment by KristofferC** [comment]
I think this has been fixed by https://github.com/JuliaLang/julia/pull/39935 now.

**Pull request closed without merging**